code/game/objects/items/weapons/defib.dm

```diff
@@ -381,6 +381,8 @@
 								busy = 0
 								update_icon()
 								return
+					if(H.heart_attack)
+						H.heart_attack = 0
```

Comment:
Due to the placement of this, it will jolt your heart back if your chest is exposed. I'll add a notification message for the user if a heart attack is started.